brain/task_executor.py

```
@@ -845,90 +1017,72 @@ async def _execute_user_plugin(self, task_id: str, up_decision: Any) -> TaskResu
+            run_body: Dict[str, Any] = {
+                "task_id": task_id,
+                "plugin_id": plugin_id,
+                "entry_id": plugin_entry_id or "run",
+                "args": safe_args,
+            }
+
+            timeout = httpx.Timeout(10.0, connect=2.0)
+            async with httpx.AsyncClient(timeout=timeout) as client:
+                r = await client.post(runs_endpoint, json=run_body)
+                if not (200 <= r.status_code < 300):
+                    raise RuntimeError(f"/runs returned {r.status_code}: {r.text}")
```

Ruff (0.14.11)

Abstract `raise` to an inner function

(TRY301)